Formula/faas-cli.rb

```diff
@@ -0,0 +1,35 @@
+class FaasCli < Formula
+  desc "CLI for templating and/or deploying FaaS functions"
+  homepage "http://docs.get-faas.com/"
+  url "https://github.com/alexellis/faas-cli/archive/0.3.tar.gz"
+  sha256 "d4db7ddb0ea1dadf469c3da9893b004874d19570d09457e8cf86e5aef0df00f5"
+
+  depends_on "go" => :build
+  depends_on "docker" => :build
```

Comment:
(`:recommended` automatically creates a `--without-docker` option for the formula)